### PR TITLE
DEVDOCS-2496: Update PriceLists filters

### PR DESCRIPTION
**Pull request by ektravel**

### CODE REVIEW

**Review by aglensmith — APPROVED** [review]

File: reference/price_lists.v3.yml [hunk 1]
```diff
@@ -2,179 +2,86 @@ swagger: '2.0'
...
 
     ## Price Lists
 
+    The association of a Price List to a Customer Group can be done either via the Control Panel or using the [Customer Groups API.](/api-reference/customer-subscribers/customers-api)
 
-    The association of a Price List to a Customer Group can be done either via
-    the Control Panel or using the [Customer Groups
-    API.](/api-reference/customer-subscribers/customers-api)
-
+    Additionally, [Price List Assignments](https://developer.bigcommerce.com/api-reference/store-management/price-lists/price-lists-records/) can be created to assign Price Lists to a specific [Channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api). Price lists assigned to a channel apply to all shoppers on that channel, unless there is a more specific assignment.
```
Comment (megdesko) [inline reference/price_lists.v3.yml:35]: or there are customer group discounts set up for the shopper's customer group

File: reference/price_lists.v3.yml [hunk 2]
```diff
@@ -2,179 +2,86 @@ swagger: '2.0'
...
-    **IF** `Price list` assigned to current `Customer Group` **AND** `Price
-    List` assigned to current `Channel`:
-
-    * Use this Price List -- any prices not found fall back to the catalog price
-    (or in the case of multi-currency, auto-converted prices)
-
+    **IF** `Price list` assigned to current `Customer Group` **AND** `Price List` assigned to current `Channel`:
+    * Use this Price List -- any prices not found fall back to the catalog price (or in the case of multi-currency, auto-converted prices)
 
     **ELSE IF**: `Price List` assigned to current `Channel`:
```
Comment (megdesko) [inline reference/price_lists.v3.yml:52]: I think this needs to be removed- the order is 
cg + channel
cg
cg discounts
channel price list

File: reference/price_lists.v3.yml [hunk 3]
```diff
@@ -2,179 +2,86 @@ swagger: '2.0'
...
+    * Fall back to the catalog price (or in the case of multi-currency, auto-converted prices)
 
-    * Fall back to the catalog price (or in the case of multi-currency,
-    auto-converted prices)
-
-
-    **Note:** Price Lists cannot be assigned to a customer group that has custom
-    group discounts -- the customer group discounts must be deleted first.
-
+    **Note:** Price Lists cannot be assigned to a customer group that has custom group discounts -- the customer group discounts must be deleted first.
```
Comment (megdesko) [inline reference/price_lists.v3.yml:67]: `has customer group discounts`

File: reference/price_lists.v3.yml [hunk 4]
```diff
@@ -2,179 +2,86 @@ swagger: '2.0'
...
-
-    - There are no direct webhooks available for Price Lists. Since Price Lists
-    directly relate to products, webhooks related to products will fire for
-    corresponding changes such as pricing.
-
+    - Price Lists cannot be assigned to a customer group that has custom group discounts -- the customer group discounts must be deleted first.
+    - Bulk pricing Tiers may additionally be associated with a price record to indicate different pricing as the quantity in cart increases.
+    - If a variant has a `Price Record` any existing product-level bulk pricing will not apply in the cart. For variants without `Price Records`, any existing product bulk pricing will apply.
+    - Price Lists Records accepts bulk upsert. Only one [Bulk upsert](https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-records/setpricelistrecordcollection) can done at a time. Running more than one in parallel on the **same store** will cause a 429 error and the request will fail.
+    - There are no direct webhooks available for Price Lists. Since Price Lists directly relate to products, webhooks related to products will fire for corresponding changes such as pricing.
```
Comment (megdesko) [inline reference/price_lists.v3.yml:74]: There are no webhooks at all for price lists (we have an epic to do this at some point)
If catalog prices are updated, product webhooks fire.

**Pull request merged**